Clockify.Net/Models/Users/UserDto.cs

```diff
@@ -14,5 +14,6 @@ public class UserDto
         public string ProfilePicture { get; set; }
         public UserSettingsDto Settings { get; set; }
         public UserStatus? Status { get; set; }
+        public List<UserRolesDto> Roles { get; set; }
```

Comment:
Please don't forget to initialize this collection!
```suggestion
        public List<UserRolesDto> Roles { get; set; } = [];
```